maven-config/pom.xml

```diff
@@ -211,5 +200,47 @@
       </plugins>
     </pluginManagement>
   </build>
+  
+  <profiles>
+    <profile>
+      <id>central.snapshot</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.central</groupId>
+            <artifactId>central-publishing-maven-plugin</artifactId>
+            <version>0.7.0</version>
+            <extensions>true</extensions>
+            <configuration>
+              <publishingServerId>central.snapshots</publishingServerId>
+              <autoPublish>true</autoPublish>
+              <waitUntil>published</waitUntil>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>central.release</id>
+      <properties>
+        <autoRelease>false</autoRelease>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.central</groupId>
+            <artifactId>central-publishing-maven-plugin</artifactId>
+            <version>0.7.0</version>
+            <extensions>true</extensions>
+            <configuration>
+              <publishingServerId>central.releases</publishingServerId>
+              <autoPublish>${autoRelease}</autoPublish>

```

Comment:
enforces confirmation for hot releases. therefore we can conveniently re-run ... deployments and don't need to be afraid of running the release pipeline early on. 
![central-deployments](https://github.com/user-attachments/assets/8ccdeefb-c904-49f4-8c7b-680eb9a73225)